vmdb/spec/models/ems_refresh/refreshers/foreman_refresher_spec.rb

```diff
@@ -25,14 +25,11 @@
 
     VCR.use_cassette("#{described_class.name.underscore}_api_v2") do
       EmsRefresh.refresh(provider)
-
-      configuration_manager.reload
-      expect(configuration_manager.last_refresh_error).to be_nil
-
-      provider.provisioning_manager.reload
-      expect(provisioning_manager.last_refresh_error).to be_nil
     end
 
+    expect(configuration_manager.reload.last_refresh_error).to be_nil
+    expect(provisioning_manager.reload.last_refresh_error).to be_nil
```

Comment:
This needs to be in the cassette block.  Otherwise the cassette will raise the missing http interactions failure at the end of the block.